parm/archive/gefs.yaml.j2

```diff
@@ -0,0 +1,30 @@
+{% set cycle_HH = current_cycle | strftime("%H") %}
+{% set cycle_YMD = current_cycle | to_YMD %}
+{% set cycle_YMDH = current_cycle | to_YMDH %}
+{% set head = RUN + ".t" + cycle_HH + "z." %}
+
+gefs:
+    name: "GEFS"
+    target: "{{ ATARDIR }}/{{ cycle_YMDH }}/gefs.tar"
+    required:
+        {% set file_set = [] %}
+
+        {% set tmpl_dict = ({ '${ROTDIR}':ROTDIR,
+                              '${RUN}':RUN,
+                              '${YMD}':cycle_YMD,
+                              '${HH}':cycle_HH,
+                              '${GRID}': '0p50',
+                              '${MEMDIR}': 'mem000' }) %}
+
+        {% set COMIN_ATMOS_GRIB_1p00 = COM_ATMOS_GRIB_GRID_TMPL | replace_tmpl(tmpl_dict) %}
+
+        # Select ensstat files to copy to the arcdir
+        {% if RUN == "gefs" %}
+            {% if path_exists(COMIN_ATMOS_GRIB_1p00) %}
+                {% for fhr in range(FHMIN_GFS, FHMAX_GFS + FHOUT_GFS - 70, FHOUT_GFS) %}
```

Comment:
@AntonMFernando-NOAA I was able to successfully archive pgrb data to HPSS when I removed `- 70` from this line. I was wondering why you subtracted 70 here?